### PR TITLE
[LWG motion 24] P2697R1 Interfacing bitset with string_view

### LINKED ISSUE #6310: [2023-06 LWG Motion 24] P2697R1 Interfacing bitset with string_view
https://wiki.edg.com/pub/Wg21varna/StrawPolls/P2697R1.pdf

### PR DESCRIPTION
**Pull request by burblebee**
Fixes #6310.
Fixes cplusplus/papers#1387

### CODE REVIEW

File: source/utilities.tex [hunk 1]
```diff
@@ -9625,10 +9625,18 @@
           = basic_string<charT, traits, Allocator>::npos,
         charT zero = charT('0'),
         charT one = charT('1'));
+    template<class charT, class traits>
+      constexpr explicit bitset(
+        basic_string_view<charT, traits> str,
+        typename basic_string_view<charT, traits>::size_type pos = 0,
```
Comment (JohelEGP) [inline source/utilities.tex:9631]: Why not use `std::size_t` directly?

File: source/utilities.tex [hunk 2]
```diff
@@ -9761,6 +9769,14 @@
       = basic_string<charT, traits, Allocator>::npos,
     charT zero = charT('0'),
     charT one = charT('1'));
+template<class charT, class traits>
+ constexpr explicit bitset(
+  basic_string_view<charT, traits> str,
```
Comment (JohelEGP) [inline source/utilities.tex:9774]:
```suggestion
  constexpr explicit bitset(
    basic_string_view<charT, traits> str,
```

**Review by JohelEGP — APPROVED** [review]
I'm using <https://isocpp.org/files/papers/P2697R1.pdf>.

File: source/utilities.tex (hunk 2 above)
Comment (burblebee) [reply source/utilities.tex:9774]: done, thanks!

**Review by JohelEGP — APPROVED** [review]

File: source/utilities.tex (hunk 1 above)
Comment (burblebee) [reply source/utilities.tex:9631]: A question for the authors...

**Pull request merged**